server/views/components/work-media/work-media.njk

```diff
@@ -0,0 +1,13 @@
+<div class="row bg-black work-media js-work-media">
+  <div class="work-media__controls js-work-media-controls pointer-events-none">
+    {% if model.queryString %}
+      <div class="flush-grid-left {{ {s:2} | spacingClasses({padding: ['top']}) }}">
+        {% componentV2 'return-to-results', {queryString: model.queryString}, null, {extraClasses: ['js-work-media-control', 'pointer-events-all']} %}
```

Comment:
Where else do we use this?
Can we not jut stick the classes on the component?

Having such a configurable component will make it very hard to change / update / iterate on in the future.

We already have a few components that have been overloaded with configuration.